interfaces/cython/cantera/utils.pyx

```diff
@@ -72,3 +72,78 @@ class CanteraError(RuntimeError):
     pass
 
 cdef public PyObject* pyCanteraError = <PyObject*>CanteraError
+
+cdef anyvalueToPython(string name, CxxAnyValue& v):
```

Comment:
names in `utils.pyx` are not very 'pythonic' (although it's not exposed to Python) - `anyvalue_to_python` would be (same for `anymap_to_dict` below - would suggest `dict` as this is what it returns)